login_server/loginserver.py

```diff
@@ -93,14 +93,20 @@ def run(self):
                     else:
                         raise
 
+    def game_servers_for_current_mode(self, game_setting_mode: str):
```

Comment:
Were you planning to use this somewhere?

Reply:
Removing for now. Was previously used to only show relevant servers, but we're now showing them all.